src/components/pages/AboutPage/TeamMember.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '../../../utils/fontAwesome';
+import '../../../css/pages/AboutPage.css';
+
+class TeamMember extends React.Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    favMovie: PropTypes.string.isRequired,
+    favShow: PropTypes.string.isRequired,
+    favGenre: PropTypes.string.isRequired,
+    imgUrl: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showInfo: false
+    };
+  }
+
+  showInfo = () => {
```

Comment:
This and the `hideInfo` functions can be condensed into just a toggle function.

```suggestion
  toggleInfoVisible = () => this.setState(prevState => ({ showInfo: !prevState.showInfo }));
```